go/engine/login_provision.go

```diff
@@ -145,9 +145,8 @@ func (e *loginProvision) Run(ctx *Context) error {
 	// check to make sure local files stored correctly
 	e.verifyLocalStorage()
 
-	go func() {
-		stellar.InitWalletSoft(context.Background(), e.G())
-	}()
+	// initialize a stellar wallet for the user if they don't already have one.
+	stellar.InitWalletSoft(context.Background(), e.G())
```

Comment:
InitWalletSoft will always get blocked by the sigchain guard this way. The guard is cleared in a defer on line 89 above. Could just clear the guard right before. Double clear is fine, it's just an atomic bool. (The comment on in sigchain_guard.go is a whole essay, I don't know why I didn't realize it's just a bool)

Reply:
ok, cleared